### ISSUE TITLE
Rakudo™ 

### ISSUE THREAD

**Issue by coke** [Raku/raku.org#192]
[Rakudo™](https://rakudo.org/) is found in the raku.org site, but i don't think Rakudo is trademarking itself, only Raku. We should find all instances of this and remove it.

Found on https://raku.org/community/

**Comment by coke** [comment]
Looks like it was added in 6607b9c7

**Comment by lizmat** [comment]
It's my understanding that both got trademarked by TPRF.  @codesections ??

**Comment by codesections** [comment]
tldr; "Rakudo™" is correct.

I went back and checked the emails and we have not registered the "Rakudo" trademark.  But registration isn't required in order to have a valid trademark (though it makes enforcing it easier).  But the attorney who registered the "Raku" mark suggested that we use ™ with "Rakudo" to establish its status as a not-yet-registered mark.  (E.g., when you register a trademark, the registration is effective back to the first "use in commerce" of the trademark – and using "Rakudo™" is pretty good evidence that we're using the trademark.

Incidentally, "Camelia™" is in the same used-in-commerce-but-not-registered situation.
